src/view/pie/PieChart.svelte

```diff
@@ -0,0 +1,111 @@
+<script>
+	import { onMount, onDestroy, getContext } from "svelte";
+
+	import Chart from "chart.js/auto";
+
+	import PieChartLegend from "./PieChartLegend.svelte";
+
+	const spreadsheetStore = getContext("spreadsheetStore");
+	const { statuses, pieData } = spreadsheetStore.stores;
+
+	let chart = null, canvasEl;
+
+	$: if (chart) {
+		chart.data = $pieData;
+		chart.update();
+	}
+
+	$: if (chart) { updateDataVisibility($statuses); }
+
+	/**
+	 * Update chart data visibility based on statuses store entries.
+	 *
+	 * @param {StatusEntry[]}	statusEntries -
+	 */
+	function updateDataVisibility(statusEntries) {
+		for (let cntr = 0; cntr < statusEntries.length; cntr++) {
+			chart[statusEntries[cntr].value ? 'show' : 'hide'](0, cntr);
+		}
+	}
+
+	onMount(() => {
+		chart = new Chart(canvasEl, {
+			type: "pie",
+			data: $pieData,
+			options: {
+				aspectRatio: 1,
+				borderColor: "transparent",
+				layout: { padding: 0 },
+				maintainAspectRatio: false,
+				plugins: {
+					legend: { display: false },
+					tooltip: { enabled: false }
+				},
+				responsive: false
```

Comment:
Why are tooltips and responsiveness disabled?

Reply:
Chart.js unfortunately doesn't have an easy way to make DOM / HTML tooltips.  Refer to this issue for conversation: https://github.com/chartjs/Chart.js/issues/7195

There is a custom legend provided in my changes that is much more capable than the legend generated by `chart.js`. I assume you have at least tried to actually run the app and see what it looks like at this point. 

It's sort of possible to create your own HTML tooltips, but not worth it from the amount of code it would require and in this case the scant bit of info actually provided by those tooltips are better represented in the custom legend which gives a real time count of the different status categories. 

There is no way to control the separation between the chart and legend with responsive mode enabled and it just doesn't look good or provide a good UX. 

It's certainly possible to add in HTML tooltips with more code, but the custom legend really does well. 

Also if you have run the app and checked it out you can see that you can click on the graph and it now sets the category exclusively as well as the legend being clickable and in this case you can actually know the chart & legend is clickable.